### PR TITLE
[FLTK] Update to 1.3.8

### PR DESCRIPTION
**Pull request by gvnnz**
Updates FLTK to version 1.3.8. Opened as draft because of point 2 (see below).

- #### What does your PR fix?  
Nothing, just updates the port.

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?  
Tested on Linux. Proper testing is needed on other platforms. 

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?  
 Yes.

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?  
Yes.

### CODE REVIEW

**Review by JonLiu1993 — CHANGES_REQUESTED** [review]

File: ports/fltk/vcpkg.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "fltk",
-  "version": "1.3.7",
+  "version": "1.3.8",
   "port-version": 1,
```
[the remark this answers is not in the document]
Comment (gvnnz) [reply ports/fltk/vcpkg.json]: Done.

File: ports/fltk/fltk_version.dat [hunk 2]
```diff
@@ -1 +1 @@
-1.3.7
```
[the remark this answers is not in the document]
Comment (gvnnz) [reply ports/fltk/fltk_version.dat:1]: No idea honestly: I thought this was something required by vckpg. I removed it and FLTK installed just fine.

**Comment by JonLiu1993** [comment]
@gvnnz ,Please run command "./vcpkg x-add-version fltk --overwrite-version" and commit again

**Comment by gvnnz** [comment]
> ports/fltk/fltk_version.dat

Done.

File: versions/f-/fltk.json [hunk 3]
```diff
@@ -1,5 +1,15 @@
...
+    {
+      "git-tree": "a8d08572b3cf440fb5f587f87d5fd04ba34f8aec",
+      "version": "1.3.8",
+      "port-version": 0
+    },
+    {
+      "git-tree": "f00f64e54ad71979c34d72f062c9fd2c01236051",
+      "version": "1.3.8",
+      "port-version": 1
+    },
```
Comment (JonLiu1993) [inline versions/f-/fltk.json:12]:
```suggestion
```

Comment (JonLiu1993) [reply versions/f-/fltk.json:12]: I don't have permission to modify this for you, please run the command "./vcpkg x-add-version fltk --overwrite-version" and commit again after the modification is complete

**Review by JonLiu1993 — CHANGES_REQUESTED** [review]

File: versions/f-/fltk.json [hunk 4]
```diff
@@ -1,5 +1,15 @@
...
+    {
+      "git-tree": "0a6f2dbd574e17da82e99414a644fca8f8efce52",
+      "version": "1.3.8",
+      "port-version": 0
+    },
+    {
+      "git-tree": "f00f64e54ad71979c34d72f062c9fd2c01236051",
+      "version": "1.3.8",
+      "port-version": 1
+    },
```
Comment (JonLiu1993) [inline versions/f-/fltk.json:12]:
```suggestion
```

Comment (JonLiu1993) [reply versions/f-/fltk.json:12]: We only need to add git-tree once per merge, so that we can go back to our historical version later

**Review by JonLiu1993 — CHANGES_REQUESTED** [review]

**Review by JonLiu1993 — APPROVED** [review]

**Comment by strega-nil-ms** [comment]
Thanks!

**Pull request merged**